backend/src/modules/doubles-tournament/presentation/rest-api/DoublesTournamentRouter.ts

```diff
@@ -48,7 +53,33 @@ export function doublesTournamentRouter(
     return response.status(StatusCodes.OK).json(new MatchesQueueDto(queryResult.tournamentId.raw, queuedMatchesDto));
   };
 
+  const getAllTournaments = async (request: Request, response: Response) => {
+    const queryResult = await queryPublisher.execute<FindAllDoublesTournamentsResult>(new FindAllDoublesTournaments());
+    if (!queryResult) {
+      return response.status(StatusCodes.NOT_FOUND).json({ message: "There aren't any tournaments ready to start" });
+    }
+    return response.status(StatusCodes.OK).json(new TournamentListDto(queryResult.map(toTournamentDto)));
+  };
+
+  const startTournament = async (request: Request, response: Response) => {
+    const requestBody: PostTournamentStartRequestBody = request.body;
```

Comment:
@nowakprojects a jeszcze zastanawiałem się nad tym, czy w tym POST jest sens dawać body, skoro w endpoincie mamy już tournamentId:
czy wystarczy dać puste body i 
`const { tournamentId } = request.params;`

Reply:
po prostu nie mamy body, na to wygląda.